pre-reg-113-115-application-default.properties

```diff
@@ -0,0 +1,86 @@
+#helm chart properties
+#db.source.dbuser.password
+#db.destination.dbuser.password
+#server.port
+#mosip.base.url
+#mosip.prereg.client.secret
+#encryptBaseUrl
+#decryptBaseUrl
+#source.object.store.secretkey
+#destination.object.store.secretkey
+#isNewDatabase=true
+
+#Database config
+mosip.source.database.hostname=postgres-postgresql.postgres
+mosip.source.database.port=5432
+source.db.name=mosip_prereg_qa3
+# primary database connection
+datasource.primary.jdbcUrl=jdbc:postgresql://${mosip.source.database.hostname}:${mosip.source.database.port}/${source.db.name}
+datasource.primary.username=postgres
+datasource.primary.password=${db.source.dbuser.password}
+datasource.primary.driver-class-name=org.postgresql.Driver
+
+mosip.destination.database.hostname=postgres-postgresql.postgres
+mosip.destination.database.port=5432
+destination.db.name=mosip_prereg_qa_upgrade
+# secondary database connection
+datasource.secondary.jdbcUrl=jdbc:postgresql://${mosip.destination.database.hostname}:${mosip.destination.database.port}/${destination.db.name}
+datasource.secondary.username=postgres
+datasource.secondary.password=${db.destination.dbuser.password}
+datasource.secondary.driver-class-name=org.postgresql.Driver
+
+## Hibernate Properties
+# The SQL dialect makes Hibernate generate better SQL for the chosen database
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+
+spring.main.web-application-type=none
+
+server.port=8081
+
+mosip.base.url=https://qa3.mosip.net
+cryptoResource.url=${mosip.base.url}/v1/keymanager
+
+appId=prereg
+clientId=mosip-prereg-client
+secretKey=${mosip.prereg.client.secret}
+
+decryptAppId=REGISTRATION
+decryptReferenceId=
+encryptAppId=PRE_REGISTRATION
+encryptReferenceId=INDIVIDUAL
+
+decryptBaseUrl=${mosip.base.url}
+encryptBaseUrl=https://qa-upgrade.mosip.net
+
+# Hibernate ddl auto (create, create-drop, validate, update)
+spring.jpa.hibernate.ddl-auto = update
+
+spring.main.allow-circular-references=true
+spring.cloud.compatibility-verifier.enabled=false
```

Comment:
Above 3 properties do we need in the external propterties, can it stay internal ?

Reply:
sure moved it to internal.